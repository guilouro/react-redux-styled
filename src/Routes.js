import React from 'react';
import { HashRouter, Switch, Route } from 'react-router-dom';
import Main from './components/Main';

const Routes = () => (
    <HashRouter>
        <Switch>
            <Route path="/" exact={true} component={Main} />
        </Switch>
    </ HashRouter>
)

export default Routes;
