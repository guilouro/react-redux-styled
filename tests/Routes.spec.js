import React from 'react';
import { shallow } from 'enzyme';
import Routes from '../src/Routes';

describe('<Routes />', () => {
    it('Should render correctly', () => {
        const wrapper = shallow(<Routes />);
        expect(wrapper).toMatchSnapshot();
    });
});
