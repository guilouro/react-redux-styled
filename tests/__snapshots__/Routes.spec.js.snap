// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Routes /> Should render correctly 1`] = `
<HashRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
  </Switch>
</HashRouter>
`;
